lesson4/main.js

```diff
@@ -0,0 +1,27 @@
+
+const ul = document.getElementById('js-ul');
+const frag = document.createDocumentFragment();
+
+const items = [
+  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+] 
+
+items.forEach(item => {
+  const li = document.createElement('li');
+  const anchor = document.createElement('a');
+  const image = document.createElement('img');
+
+  anchor.href = '/' + item.to;
```

Comment:
Nit

you can also write blow.   
```suggestion
  anchor.href = `/${item.to}`;
```

Reply:
Thank you for your review.
I'm learning.
I fixed it as soon as possible.